src/garage/tf/models/gru.py

```diff
@@ -28,10 +31,10 @@ def gru(name,
             hidden state is trainable.
 
     Return:
-        outputs (tf.Tensor): Entire time-series outputs.
-        output (tf.Tensor): Step output.
-        hidden (tf.Tensor): Step hidden state.
-        hidden_init_var (tf.Tensor): Initial hidden state.
+        tf.Tensor: Entire time-series outputs, with shape :math:`(N, T, S^*)`.
+        tf.Tensor: Step output, with shape :math:`(N, S^*)`.
+        tf.Tensor: Step hidden state, with shape :math:`(N, hidden_dim)`
```

Comment:
please don't use strings (e.g. `hidden_dim`) as shape names. These don't render properly in the math environment. Pick a symbol such as H or Z.